snuba/clickhouse/columns.py

```diff
@@ -28,7 +39,7 @@ def for_schema(self) -> str:
     @staticmethod
     def to_columns(
         columns: Sequence[Union[Column, Tuple[str, ColumnType]]]
-    ) -> Sequence[Column]:
+    ) -> List[Column]:
```

Comment:
Why did this change? Is this mutated somewhere later? Should/can this be a `MutableSequence` instead of a `List`?

Reply:
I thought I tried MutableSequence but it did not like concatenating sequences with the + operator, though I tried again this morning and it seemed fine, Let me check again.

Reply:
Indeed, if I change it to MutableSequence I get 
```
Unsupported left operand type for + ("MutableSequence[Column]")
```
at line 408:
```
return ColumnSet(self.columns + other.columns)
```

Reply:
I think that is actually a good error, it could easily be avoided by changing that to

```python
return ColumnSet([*self.columns, *other.columns])
```

which I think is a more accurate reflection of the intent there (clearly returning a new list), possibly?

Reply:
I guess my expectation of a + operator on two lists is to return a third list with the concatenation of the two. I suspect the + operator is not declared by mypy on Sequence because only List supports it more than for semantics reasons.
Anyway I am ok, using your expression. Will send a followup PR with the change.